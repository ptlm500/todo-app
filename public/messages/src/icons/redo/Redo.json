[
  {
    "id": "todo.icons.redo.title",
    "description": "Redo icon title",
    "defaultMessage": "Redo"
  }
]