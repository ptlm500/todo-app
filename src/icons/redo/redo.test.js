import React from 'react';
import renderer from 'react-test-renderer';
import Redo from './Redo';

describe('Redo', () => {
  it('renders a Redo icon', () => {
    const tree = renderer.create(<Redo />);
    expect(tree).toMatchSnapshot();
  });

  it('renders a Redo icon with custom title', () => {
    const tree = renderer.create(<Redo title={'test title'}/>);

    expect(tree.root.findByType('title').children).toEqual(['test title']);
    expect(tree).toMatchSnapshot();
  });
});
