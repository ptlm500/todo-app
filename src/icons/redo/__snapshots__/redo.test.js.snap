// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redo renders a Redo icon 1`] = `
<svg
  height="9px"
  version="1.1"
  viewBox="0 0 21 9"
  width="21px"
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Redo
  </title>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <g
      transform="translate(-747.000000, -1533.000000)"
    >
      <g
        transform="translate(100.000000, 1428.000000)"
      >
        <g
          id="Outlined-/-Content-/-redo"
          transform="translate(646.000000, 98.000000)"
        >
          <g>
            <polygon
              id="Path"
              points="0 0 24 0 24 24 0 24"
            />
            <path
              d="M18.4,10.6 C16.55,8.99 14.15,8 11.5,8 C6.85,8 2.92,11.03 1.54,15.22 L3.9,16 C4.95,12.81 7.95,10.5 11.5,10.5 C13.45,10.5 15.23,11.22 16.62,12.38 L13,16 L22,16 L22,7 L18.4,10.6 Z"
              fill="currentColor"
            />
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;

exports[`Redo renders a Redo icon with custom title 1`] = `
<svg
  height="9px"
  version="1.1"
  viewBox="0 0 21 9"
  width="21px"
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    test title
  </title>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <g
      transform="translate(-747.000000, -1533.000000)"
    >
      <g
        transform="translate(100.000000, 1428.000000)"
      >
        <g
          id="Outlined-/-Content-/-redo"
          transform="translate(646.000000, 98.000000)"
        >
          <g>
            <polygon
              id="Path"
              points="0 0 24 0 24 24 0 24"
            />
            <path
              d="M18.4,10.6 C16.55,8.99 14.15,8 11.5,8 C6.85,8 2.92,11.03 1.54,15.22 L3.9,16 C4.95,12.81 7.95,10.5 11.5,10.5 C13.45,10.5 15.23,11.22 16.62,12.38 L13,16 L22,16 L22,7 L18.4,10.6 Z"
              fill="currentColor"
            />
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;
